src/index.js

```diff
@@ -69,12 +69,15 @@ var sleepRepository;
 var activityRepository;
 var user;
 
-randomId = Math.floor(Math.random() * (50 - 1) + 1);
-userRepository = new UserRepository(userData, randomId);
-hydrationRepository = new HydrationRepository(hydrationData, randomId);
-sleepRepository = new SleepRepository(sleepData, randomId);
-activityRepository = new ActivityRepository(randomId, activityData);
-user = new User(userRepository.getUserData());
+Promise.all([userData]).then(element => {
```

Comment:
Just going to drop this comment here to make sure I understand this correctly, the `Promise.all([userData])` is the information we want to load, which means we need the sleepRepo, hydrationRepo, etc... into the array so that we can properly populate the information on the DOM?

Reply:
@VeeAndrade you got it! The DOM loading functions should not run until all promises passed in here are resolved